using-mobile-extensions/adobe-analytics/analytics-api-reference.md

```diff
@@ -554,6 +554,9 @@ ACPAnalytics.getQueueSizeWithCompletionHandler { (queueSize, error) in
 {% endtabs %}
 
 ## getTrackingIdentifier <a id="gettrackingidentifier"></a>
+{% hint style="warning" %}
+Analytics extension will no longer generate/retrieve a new tracking identifier (AID) for new visitors. For the visitors which have an AID value previously generated will continue retrieve AID value with this API. New users will use the ECID value as the primary identity [Identity](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/identity#visitor-ids-in-apps).
+{% endhint %}
 
 {% hint style="warning" %}
 Before you use this API, see the documentation on identifying [unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).
```

Comment:
I think we should consolidate the description for this API and keep only the info/warning bubbles that are strict necessary. I would suggest the following:
- move the actual API description on top 
- followed by a warning bubble which mentions starting with which version the AID will no longer be returned for new users
- can we also add the resetIdentities API and list out the cleared identities for Analytics?

Example:
```
Retrieves the Analytics tracking identifier that is generated for this app/device instance. The ID is preserved between app upgrades and is removed when the app is uninstalled, as well as on resetIdentities<relative link> or on privacy status opt out. 

{% hint style="warning" %}
Starting with vx (Android) / vy(iOS) this API does not generate or retrieve a new tracking identifier (AID) for new visitors. For the visitors which have an AID value previously generated will continue retrieve AID value with this API, and new users will use the ECID (MID) value as the primary identity.
Before using this API, see the documentation on identifying [unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html).
{% endhint %}